src/pages/runs/[date].tsx

```diff
@@ -0,0 +1,81 @@
+import Layout from "@/components/Layout";
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import { isMatch } from "date-fns";
+import { graphql, Link, PageProps } from "gatsby";
+import * as React from "react";
+
+const RunsPage: React.FC<PageProps<Queries.RunsPageQuery>> = ({
+  data,
+  params,
+}) => {
+  const drpdPages = data.allDrpdJson.edges;
+  const date = params.date;
+
+  return (
+    <Layout date={date}>
```

Comment:
Does passing date to the Layout do something?

Reply:
the layout sends the date to header and from header to date picker so it can be shown

probably should have made it an optional prop on layout though, will fix that

Reply:
Would it be better to use react context? We can set it up already for more use later

Reply:
I'm not sure how we'd do that, can you elaborate a bit more

Reply:
Check the accepted answer in this, maybe we could implement something like it. Let me know if you think it's worth it.

https://stackoverflow.com/questions/73101274/how-to-update-react-context-from-multiple-conmponents

Reply:
Since date is a url parameter, I don't know if there's a way to directly access it from the context provider component (using gatsby). 

We can do something like have { date, setDate } as context and have each page call setDate to update the date, but I don't think it's a good idea to have it as an an additional state because it can get out of sync if we don't update it properly from every page